active/0000-addon-options.md

```diff
@@ -0,0 +1,81 @@
+- Start Date: 2015-03-24
+- RFC PR:
+- ember-cli Issue:
+
+# Summary
+
+Enabling addons to specify custom prompts that would inquire the user, and perform arbitrary actions based on those responses. Actions could range from running blueprints, installing additional addons, setting configuration.
+
+Similar to what Yeoman has:
+
+![from yeoman](http://yeoman.io/static/image_4.d4f761cf89.png)
+
+# Motivation
+
+This would pave the way for users creating custom 'app' blueprints that can have their personal/comapany's workflows, with default settings, and optionals. This could include installing the addons you always install, e.g. ember-cli-less, bootstrap, font-awesome, etc.
+
+This would also make addons much more powerful, since they can ask the user critical questions, and not always assume. This would also make ember-cli more powerful, and as we draw closer to a core cli, these features will be more important for wider adoption.
+
+# Detailed design
+
+Prompts are defined at the blueprint level, so you can create
+a prompt that runs during `ember install my-addon` or during the explicit `ember g my-blueprint`.
```

Comment:
See https://github.com/ember-cli/rfcs/pull/7#issuecomment-85587635.

What about `ember install my-addon --interactive`? But then you need to know that the addon is interactive, maybe we can make it an option that can be set in the `.ember-cli` file for those that want it all the time.